playground/src/routes/workflows.index.tsx

```diff
@@ -1,6 +1,6 @@
 import { workflowsQueryOptions } from "@/lib/hooks/useWorkflows";
 import { createFileRoute } from "@tanstack/react-router";
-import { NewWorkflow } from "./workflow.new";
+import { NewWorkflow } from "./workflows.new";
 
 export const Route = createFileRoute("/workflow/")({
```

Comment:
do you want to update the route here as well? otherwise the url for the UI is `/workflow` still

Reply:
same goes for the other workflow files that were renamed

Reply:
oh forgot to run tanstack router after changing this - more changes incoming